Facebook.Android/nuget/Xamarin.Facebook.Android.nuspec

```diff
@@ -16,6 +16,7 @@
       <group targetFramework="MonoAndroid403">
```

Comment:
I am having a hard time matching up the NuGet dependency list with Xamarin.Facebook/packages.config.  Should they not be the same?